Carbon.Redis/Abstractions/Serialization.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Carbon.Caching.Abstractions
+{
+    public enum Serialization
```

Comment:
this enum's name sounds so generic that Ithought it was coming from a microsoft package :D Maybe we should also call this enum somehing like SerializationType/SerializerType